public/locales/en/common.json

```diff
@@ -136,7 +136,7 @@
   "features - sleep timer": "Sleep timer",
   "features - subscribe to listener profiles": "Subscribe to listener profiles",
   "features - subscribe to podcasts": "Subscribe to podcasts",
-  "features - support open source software": "Support open source software",
+  "features - support open source software": "Support software that lets only you listen",
```

Comment:
Synergy with https://github.com/podverse/podverse-web/pull/888/files#diff-4840952983d2a94f27deb00bf7247347345b60e3a96dc2aa41a6a50ecacdf1b6R357

Reply:
I'm sorry, @kreonjr and I just don't agree with this idea that we need to retire "open source" from our vernacular. These days I'm more inclined to think "open source" is an adequate term, and if corporations misuse it to claim something is "open source" when they might only mean "transparent," that's their problem. If a *source* is *open*, then you should be able to do anything you want with it, and the *source* should remain *open*.

We don't want to stop using the word "podcast" in the way that we mean it just because Spotify and YouTube are trying to debase it so it's indistinguishable from legacy / centralized media. "Podcast" has power and meaning in public consciousness and I don't want to let corporations totally appropriate it, and make us resort to a new alternate term people are unfamiliar with. I feel similarly about corporations calling code "open source" when their source is not actually open.

Best I can do is what we're already doing, which is to use a mixture of the terms open source, FOSS, libre, free and open source software, etc. in various places in our app and marketing text.

Reply:
Open source is used to hide copyleft, and not talk about freedoms.
It is like calling an office building a structure. If you want to get particular and have a point there, it misses said point entirely.

The word "Podcast" isn't in dispute, merely because it always what it is now.
You put it behind a paywall and censorship and now the term doesn't differentiate.

If however the uglies tried calling it "worksafe corpotransmission" in all their might of popularity,
I would similarly suggest not adopting that word.

The second you say "marketing", you evoke arbitrary concerns, of popularity, clarity, and what have you.
I ask you what you want to market, and you say "adequate".
I don't know what that is, so I can't help you.
Neither of us are good at it, which is a fair admission.
A lot of buzzwords are popular, and convey similarly nothing of value to Podverse,
be it "Killer app", "websafe software" "fullstack-ACK" "green software", or similar.

"reciprocal systems of trade" or literally any other term is bound to be more popular than
"value for value", yet there Podverse gets it right.
Using the "unpopular" term means taking part in what you build, and since it is correct that also means
users understanding the difference between types of libre software.
I posit Podverse has a very unique funding and governance model, and I think this should be communicated,
since it is a great part of why other apps are different in a way users despise.

Using a mixture of all terms, without explainging anything, _ensures_ nobody understands what is what.
"Open Source" is only good for SEO, to get hits.  It can be worked in there.

Reply:
Man, I just cannot see myself taking "open source" out  of my vocabulary. I think all language in practice has fuzziness around it. To me saying "open source" is a signal to people with a *casual* interest in free software that we are "a different, more open, and freer company than the big tech platforms," and our target audience is casual listeners. Our "marketing" model is basically relying on recommendations from podcasters who want their audience to use a Podcasting 2.0 and value-enabled app. It seems easier to me for them to say "and it's open source!" than it is for them to say "and it's copylefted libre!"

I think the specifics of software licenses are beyond what casual podcast listeners would care about, and the people who *do* care will look up specifically what license we use. The fact we provide "open source" software puts us in a small category of apps (AntennaPod and Pocket Casts are the only examples I hear of often, and Pocket Casts has a limited less-free license along the lines of what you'd like to distinguish us from).

We also don't add advertisements, and we don't sell or share private user data , but to me, the umbrella imperfect term "open source" conveys to people that we are the type of company that makes these kinds of principle-based decisions. As for the specifics, they need to look them up as it's not always easy to convey all those principles in a short phrase.

This might just be a personal failing on my part. I just still can't conceive of letting go of the word "open source." It's what I've dedicated my life to. Part of me wants to just say "sure change it to whatever you want" so that I'm not worrying about this so much, but I feel like I have to get over this gut feeling that removing the words "open source" entirely from our app and from my podcast appearances and everything related to Podverse...it's just not sitting right with me.

Maybe this should be an open topic in our [podverse/info Discussions](https://github.com/orgs/podverse/discussions) forum? I could even pin it to the top. I've heard what I've thought are compelling but opposing arguments on different sides. I really don't want to dismiss your concerns as I think they are valid. I just genuinely cannot envision a future where I'm not saying I create "open source software" and that Podverse is an "open source app."